actions/ctf-build-image/action.yml

```diff
@@ -89,21 +89,55 @@ runs:
     - name: Replace Solana deps
       if: ${{ inputs.dep_solana_sha }}
       shell: bash
-      run:
-        go get github.com/smartcontractkit/chainlink-solana@${{
-        inputs.dep_solana_sha }}
+      env:
+        DEP_SOLANA_SHA: ${{ inputs.dep_solana_sha }}
+      run: |
+        go get "github.com/smartcontractkit/chainlink-solana@${DEP_SOLANA_SHA}"
+    - name: Update plugin deps
+      shell: bash
+      env:
+        DEP_COSMOS_SHA: ${{ inputs.dep_cosmos_sha }}
+        DEP_SOLANA_SHA: ${{ inputs.dep_solana_sha }}
+        DEP_STARKNET_SHA: ${{ inputs.dep_starknet_sha }}
+        PLUGINS_MANIFEST_PATH:
+          ${{ github.workspace }}/plugins/plugins.public.yaml
```

Comment:
Is it possible this doesn't exist? I see references to this action in multiple repos: https://github.com/search?q=org%3Asmartcontractkit%20%22ctf-build-image%22&type=code

Might be worth checking if this file exists, and if it doesn't - `exit 0` ?

Reply:
https://github.com/smartcontractkit/.github/pull/1031/commits/b9dc07ab473871b4ca7e1e2d310a9b7abd46a854